README.md

```diff
@@ -69,6 +70,16 @@ the [checks documentation page](checks/checks.md).
 
 ## Usage
 
+### Docker
```

Comment:
Move this section to after `Authentication` header.

Reply:
I think it should be before because as a consumer, I would first look for a docker container to test and go with that. That was the reason to have it prior to it. 

Having a docker container makes it a lot easier for consumers to try.

Reply:
Hmm, in terms of readability it makes more sense to talk about the command line first and then explain that there is a docker option too. Also, from all the feedback we have gotten so far, everyone seems to be using the command line more frequently.

Reply:
They are using the command line option because the docker instructions were removed from README.md

Reply:
Ok, upto you.